lib/ipmi_backend_utils.pm

```diff
@@ -143,6 +143,10 @@ sub setup_console_in_grub {
         save_screenshot;
 
         if (!script_run('grep HPE /sys/class/dmi/id/board_vendor') == 0) {
+            $cmd = "sed -ri '/^terminal.*\$/ {:mylabel; n; s/^terminal.*\$//;b mylabel;}' $grub_cfg_file";
+            assert_script_run($cmd);
+            $cmd = "sed -ri '/^\$/d' $grub_cfg_file";
+            assert_script_run($cmd);
```

Comment:
Small suggestion, you could show config file content using `record_info` call would simplify investigations and make content visible in openQA web UI.

Reply:
record_info($grub_cfg_file) ?  You mean I can add this in my code or just use this for testing my code ?

Reply:
To get content we will have to call script_output, so more like `record_info("Grub config", script_output("cat $grub_cfg_file"))`. Box in the openQA won't display full content, so better to pass the content as a second parameter to display it fully.
You could add it to make changes visible when someone needs to take a look on the run. As an alternative you could also simply redirect the content to the serial port with `script_run("cat $grub_cfg_file > /dev/$serialdev")

But that's just suggestion, not a change request ;)

Reply:
Thanks. After thinking twice, I think we do not need this currently, we already upload config file as log files. And I want to keep this part short as well. But this is a good way to use record_info, I will use this when necessary in the future.